src/sets.jl

```diff
@@ -197,11 +197,11 @@ end
 dimension(s::Union{ExponentialCone, DualExponentialCone, PowerCone, DualPowerCone}) = 3
 
 """
-    PositiveSemidefiniteConeTriangle(dimension)
+    PositiveSemidefiniteConeTriangle(side_dimension)
 
-The (vectorized) cone of symmetric positive semidefinite matrices, with side length `dimension` and with off-diagonals unscaled.
+The (vectorized) cone of symmetric positive semidefinite matrices, with side length `side_dimension` and with off-diagonals unscaled.
 The entries of the upper triangular part of the matrix are given column by column (or equivalently, the entries of the lower triangular part are given row by row).
-An ``dimension \\times dimension`` matrix has ``dimension(dimension+1)/2`` lower-triangular elements, so for the vectorized cone of dimension ``n``, the corresponding symmetric matrix has side dimension ``\\sqrt{1/4 + 2 n} - 1/2`` elements.
+A ``d \\times d`` matrix has ``d(d+1)/2`` lower-triangular elements, so for the vectorized cone of dimension ``n``, the corresponding symmetric matrix has a side dimension ``d`` of ``\\sqrt{1/4 + 2 n} - 1/2`` rows/columns.
```

Comment:
Need to add "where ``d`` is `side_dimension`"

Reply:
It is pretty clear that it is when you read the whole sentence. I will repeat myself for extra clarity.